samples/sample-kotlin-android/src/main/kotlin/com/rudderstack/android/sampleapp/analytics/RudderAnalyticsUtils.kt

```diff
@@ -37,26 +35,8 @@ object RudderAnalyticsUtils {
         _rudderAnalytics?.initializeWorkManager()
     }
 
-    private fun createSecondaryInstance(
-        listener: InitializationListener?,
-        application: Application
-    ) = createInstance(
-        writeKey = WRITE_KEY_SECONDARY,
-        initializationListener = { success, message ->
-            listener?.onAnalyticsInitialized(WRITE_KEY_SECONDARY, success, message)
-        },
-        configuration = ConfigurationAndroid(
-            application = application,
-            GsonAdapter(),
-            dataPlaneUrl = DATA_PLANE_URL_SECONDARY,
-            controlPlaneUrl = CONTROL_PLANE_URL_SECONDARY,
-            trackLifecycleEvents = true,
-            recordScreenViews = true,
-        )
-    )
-
-    fun createPrimaryAnalyticsInstanceWithWorkerSupport(application: Application, listener: InitializationListener? = null): Analytics {
-        return createInstance(
+    fun createAnalyticsInstanceWithWorkerSupport(application: Application, listener: InitializationListener? = null): Analytics {
```

Comment:
I don't think this method is needed any more. 
The plain initialization is fine for now.

 ```
   fun initialize(application: Application, listener: InitializationListener? = null) {
        // add work manager support to this instance
        _rudderAnalytics = getInstance(
            writeKey = WRITE_KEY,
            initializationListener = { success, message ->
                listener?.onAnalyticsInitialized(WRITE_KEY, success, message)
            },
            configuration = ConfigurationAndroid(
                application = application,
                GsonAdapter(),
                dataPlaneUrl = DATA_PLANE_URL,
                controlPlaneUrl = CONTROL_PLANE_URL,
                trackLifecycleEvents = true,
                recordScreenViews = true,
                isPeriodicFlushEnabled = true

            )
        )
        _rudderReporter = DefaultRudderReporter(
            context = application, baseUrl = METRICS_BASE_URL, configuration = Configuration(
                LibraryMetadata(
                    name = "android",
                    sdkVersion = BuildConfig.LIBRARY_PACKAGE_NAME,
                    versionCode = BuildConfig.LIBRARY_VERSION_NAME,
                    writeKey = WRITE_KEY
                )
            ), JacksonAdapter()
        )
        _rudderAnalytics?.initializeWorkManager()
    }
```

Reply:
[feat__removes_primaryAnalytics.patch](https://github.com/rudderlabs/rudder-sdk-android/files/15355251/feat__removes_primaryAnalytics.patch)

Reply:
Simplifies and kotlinises some getters. 😁

Reply:
This is needed in order to support Work Manager feature.

Reply:
Correct, is still utilised. Check the patch 😁

Reply:
Sorry, I missed checking the patch.